src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -496,8 +496,13 @@ export default class AircraftCommander {
     runReroute(aircraft, data) {
         // TODO: is this .toUpperCase() necessary??
         const routeString = data[0].toUpperCase();
+        const readback = aircraft.pilot.applyNewRoute(routeString, aircraft.initialRunwayAssignment);
 
-        return aircraft.pilot.applyNewRoute(routeString, aircraft.initialRunwayAssignment);
+        if (readback[0]) {
```

Comment:
Let's be sure to add a comment on the _why_ here so we know next time we come to it